### ISSUE TITLE
AI infra ui - Bump patternfly-a11y tool to latest version

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/AI-infra-ui-components#77]
The patternfly-a11y tool should be bumped to version 5.2.1. If the version bump doesn't trigger any a11y failures the PR can be merged, otherwise we want to keep the PR unmerged so that we can evaluate what work needs to be done to resolve those violations.

Do not commence work on this until https://redhat.atlassian.net/browse/PF-3831 has been completed.

---

**Jira Issue:** [PF-3832](https://redhat.atlassian.net/browse/PF-3832)
